File: Python/Manual 02/Question01.py
value = input("Enter an alphabet: ")

if not ((value >= "A" and value <= "Z") or (value >= "a" and value <= "z")) or len(value) > 1:
    print("Invalid Input")
else:
    value = value.lower()

    if value == "a" or value == "e" or value == "i" or value == "o" or value == "u":
        print("Alphabet is a Vowel")
    else:
        print("Alphabet is a Consonant")

File: JavaScript/Manual 02/Question01.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter a Character: ", function (character) {
    if (!"abcdefghijklmnopqrstuvwxyz".includes(character.toLowerCase())) {
        console.log("Invalid Input");
        rl.close();
        return;
    }
    else {
        if ("aeiou".includes(character.toLowerCase()))
            console.log("The character is a Vowel.");
        else
            console.log("The character is a Consonant.");
    }

    rl.close();
});